lib/matplotlib/tests/test_legend.py

```diff
@@ -607,6 +607,21 @@ def test_legend_title_fontprop_fontsize():
     assert leg5.get_title().get_fontsize() == 20
 
 
+@pytest.mark.parametrize('alignment', ('center', 'left', 'right'))
+def test_legend_title_align(alignment):
+    fig, ax = plt.subplots()
+    ax.plot(range(10))
+    leg = ax.legend(title="Aardvark", alignment=alignment)
+    if alignment is None:
+        alignment = 'center'
+    assert leg.get_children()[0].align == alignment
+    assert leg.get_alignment() == alignment
+    for align in ['center', 'left', 'right']:
+        leg.set_alignment(align)
+        assert leg.get_children()[0].align == align
+        assert leg.get_alignment() == align
```

Comment:
This is redundant: you are running this again for each of the parametrizations. I suggest to put the test of `set_alignment` this in a separate test function instead: